tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -107,11 +109,12 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1Cluster() {
 		name      string
 		nodeRoles []nodepools.NodeRoles
 		client    *rancher.Client
+		cluster   *management.Cluster
```

Comment:
There is no need to do this. This makes the test case a bit confusing, since why would cluster be test parameter.